### PR TITLE
Release v5.0.0

### PR DESCRIPTION
**Pull request by nuttycom**

### CODE REVIEW

File: src/version.h [hunk 1]
```diff
@@ -9,7 +9,7 @@
  * network protocol versioning
  */
 
-static const int PROTOCOL_VERSION = 170050;
+static const int PROTOCOL_VERSION = 170100;
```
Comment (str4d) [inline src/version.h:12]: I confirmed that this matches the values in `src/chainparams.cpp` and [ZIP 252](https://zips.z.cash/zip-0252).

**Review by steven-ecc — APPROVED** [review]
utACK

Verified the release height and activation height.

File: doc/release-notes/release-notes-5.0.0.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
+Notable changes
+===============
+
```
Comment (str4d) [inline doc/release-notes/release-notes-5.0.0.md:3]: This release activates NU5, which I _think_ is a notable change 😄

File: src/deprecation.h [hunk 3]
```diff
@@ -10,8 +10,8 @@
 // Per https://zips.z.cash/zip-0200
 // Shut down nodes running this version of code, 16 weeks' worth of blocks after the estimated
 // release block height. A warning is shown during the 14 days' worth of blocks prior to shut down.
-static const int APPROX_RELEASE_HEIGHT = 1661960;
-static const int RELEASE_TO_DEPRECATION_WEEKS = 2;
+static const int APPROX_RELEASE_HEIGHT = 1663550;
```
Comment (str4d) [inline src/deprecation.h:13]: The mainnet block height as of this comment was 1663437, so 113 blocks before this height (which is about right for at least a few hours of CI and release work).

File: doc/release-notes/release-notes-5.0.0.md [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+The mainnet activation of the NU5 network upgrade is supported by the 4.0.0
```
Comment (str4d) [inline doc/release-notes/release-notes-5.0.0.md:4]:
```suggestion
The mainnet activation of the NU5 network upgrade is supported by the 5.0.0
```

**Review by defuse — APPROVED** [review]
utACK

**Review by str4d — APPROVED** [review]
ACK

**Review by ebfull — APPROVED** [review]

**Review by daira — APPROVED** [review]
utACK

**Pull request merged**